packages/Webkul/Admin/src/Resources/views/sales/orders/images.blade.php

```diff
@@ -7,7 +7,7 @@
 
     @foreach ($items as $item)
         @php
-            $imageCount = optional($item->product)->images->count();
+            $imageCount = optional($item->product)->images?->count();
```

Comment:
Remove the optional and use the null safe operator.

Reply:
Done Sir